### PR TITLE
Release new ouroboros-network packages

### PR DESCRIPTION
**Pull request by coot**
# Description

- Bumped versions
- Require at least network-3.1.4

# Checklist

- Branch
    - [x] Updated changelog files.
    - [x] Commit sequence broadly makes sense
    - [x] Commits have useful messages
    - [ ] The documentation has been properly updated
    - [ ] New tests are added if needed and existing tests are updated
    - [ ] If serialization changes, user-facing consequences (e.g. replay from genesis) are confirmed to be intentional.
- Pull Request
    - [x] Self-reviewed the diff
    - [x] Useful pull request description at least containing the following information:
      - What does this PR change?
      - Why these changes were needed?
      - How does this affect downstream repositories and/or end-users?
      - Which ticket does this PR close (if any)? If it does, is it [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)?
    - [x] Reviewer requested

### CODE REVIEW

File: monoidal-synchronisation/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Revision history for monoidal-synchronisation
 
+## 0.1.0.5
```
Comment (bolt12) [inline monoidal-synchronisation/CHANGELOG.md:3]: Add next version section

File: network-mux/network-mux.cabal [hunk 2]
```diff
@@ -55,8 +55,8 @@ library
                        -- The Windows version of network-3.1.2 is missing
                        -- functions, see
                        -- https://github.com/haskell/network/issues/484
-                       network         >=3.1.2.2 && <3.2,
-                       process         >=1.6 && <1.7,
+                       network        ^>=3.1.4,
+                       process        ^>=1.6,
```
Comment (bolt12) [inline network-mux/network-mux.cabal:59]: Do we need to update the network-mux version?

**Review by bolt12 — CHANGES_REQUESTED** [review]

File: network-mux/network-mux.cabal (hunk 2 above)
Comment (coot) [reply network-mux/network-mux.cabal:59]: `network-3.1.3` fixes a build failure on `Mac M1` machines, so it's good to include it. `network-3.1.4` adds `Unix Socket` support on `Windows`, not that we need it now but it's good to know.

For `process` it's just a syntactic change.

File: monoidal-synchronisation/CHANGELOG.md (hunk 1 above)
Comment (coot) [reply monoidal-synchronisation/CHANGELOG.md:9]: Done.

**Review by bolt12 — APPROVED** [review]

**Pull request merged**